From 8a6156bd26240177e710fd8623bc2eb0ef0092cf Mon Sep 17 00:00:00 2001
From: Shiji Yang <yangshiji66@outlook.com>
Date: Wed, 18 Jun 2025 22:53:22 +0800
Subject: [PATCH 10/11] MIPS: lantiq: falcon: sysctrl: add missing header
 prom.h

"prom.h" includes the prototype of ltq_soc_init(). Fix warning:

arch/mips/lantiq/falcon/sysctrl.c:185:13: error: no previous prototype for 'ltq_soc_init' [-Werror=missing-prototypes]
  185 | void __init ltq_soc_init(void)
      |             ^~~~~~~~~~~~

Signed-off-by: Shiji Yang <yangshiji66@outlook.com>
Signed-off-by: Thomas Bogendoerfer <tsbogend@alpha.franken.de>
---
 arch/mips/lantiq/falcon/sysctrl.c | 1 +
 1 file changed, 1 insertion(+)

--- a/arch/mips/lantiq/falcon/sysctrl.c
+++ b/arch/mips/lantiq/falcon/sysctrl.c
@@ -14,6 +14,7 @@
 #include <lantiq_soc.h>
 
 #include "../clk.h"
+#include "../prom.h"
 
 /* infrastructure control register */
 #define SYS1_INFRAC		0x00bc
